backend/data/admin.py

```diff
@@ -4,64 +4,112 @@
 
 class PersonNameAdmin(admin.StackedInline):
     model = models.PersonName
-    fields = ['value', 'certainty', 'note']
-
+    fields = ["value", "certainty", "note"]
```

Comment:
My editor automatically uses Black. I could turn this off for this project, but it may not be a bad idea to agree on a formatter.

Reply:
I personally don't mind formatting inconsistencies, but if you prefer we use the same formatter, I can install it.

Reply:
What I _do_ mind is (auto-)formatting an entire file when you're changing a few lines of content. It makes the review more difficult, as meaningful changes get cluttered by meaningless ones.

Anyway, we can choose something in our next meeting. It's a good idea to decide on this early in the project.

Reply:
Agreed! In the meantime, I'll turn off Black for this project to avoid large meaningless diffs.